_DeploymentAndDistroPackaging/ansible/cleanup/ciao.yml

```diff
@@ -17,16 +17,11 @@
   connection: local
   tasks:
     - name: Remove local files
-      local_action: file path={{ item }} state=absent
-      with_items:
-        - ./openrc
-        - ./admin-ciaorc
-        - ./demo-ciaorc
-        - ./clouds.yaml
-        - ./certificates
-        - ./images/.cnci_ready
-        - ./fetch
-        - /tmp/go
+      file: path={{ item }} state=absent
+      with_fileglob:
+        - /tmp/go/*
```

Comment:
So then a malicious user can pre-populate the world writable /tmp with bad data, and the ansible scripts will not overwrite the bad data and will use the bad data.